src/kiwixlistwidget.cpp

```diff
@@ -0,0 +1,113 @@
+#include "kiwixlistwidget.h"
+#include <QKeyEvent>
+#include <QToolTip>
+#include <QCursor>
+#include <QDebug>
+#include "kiwixapp.h"
+#include "klistwidgetitem.h"
+
+KiwixListWidget::KiwixListWidget(QWidget *parent)
+    : QListWidget(parent), currRow(0), m_visibleItems(count())
+{
+    connect(this, &KiwixListWidget::currRowChanged, this, &KiwixListWidget::handleCurrRowChange);
+    setMouseTracking(true);
+}
+
+void KiwixListWidget::showEvent(QShowEvent *event)
+{
+    QListWidget::showEvent(event);
+}
```

Comment:
Is this really necessary ?

Reply:
Wasn't, was a consequence of overlooking after a rebase